<launch>
    <node name="control_line_node" pkg="control_line" type="control_line_node" ns="green_robot" output="screen">
	    <param name="topic_cmd_vel" value="cmd_vel"/>
		<param name="topic_sensors_line" value="sensors/line"/>
		
        <param name="linear_speed" value="0.5"/>
        <param name="Kp" value="1.0"/>
    </node>
</launch>
